source/change-streams/tests/README.rst

```diff
@@ -0,0 +1,100 @@
+.. role:: javascript(code)
+  :language: javascript
+
+==================
+Change Streams
+==================
+
+.. contents::
+
+--------
+
+Introduction
+============
+
+The YAML and JSON files in this directory are platform-independent tests that
+drivers can use to prove their conformance to the Change Streams Spec.
+
+Several prose tests, which are not easily expressed in YAML, are also presented
+in this file. Those tests will need to be manually implemented by each driver.
+
+Test Format
+===========
+
+Each YAML files has the following keys:
+
+- ``database_name``: The default database
+- ``collection_name``: The default collection
+- ``defaults``: Default values for each test
+- ``tests``: An array of tests that are to be run independently of each other.
+  Each test will have some of the following fields:
+
+  - ``description``: The name of the test.
+  - ``skip``: Notes that this test is incomplete and should not be run
+  - ``minServerVersion``: The minimum server version to run this test against. If not present, assume there is no minimum server version.
+  - ``maxServerVersion``: Reserved for later use
+  - ``failPoint``: Reserved for later use
+  - ``topology``: The server topology/topologies to run the test against.
+    Valid topologies are ``single``, ``replicaset``, and ``sharded``.
+    Can be either a single topology, or a list of topologies.
```

Comment:
Easier to consume if this is always a list.

Reply:
done